default.nix

```diff
@@ -75,5 +69,14 @@ let
                     --prefix PATH : "${pkgs.lib.makeBinPath iohk-ops-extra-runtime-deps}"
                   '';
                });
+
+    #mkDerivation = args: super.mkDerivation (args // {
+    #enableLibraryProfiling = false;
+    #});
+  };
+});
+  cabal2nixpkgs = rec {
```

Comment:
Why has `iohk-ops` been left out? It uses `compiler.callPackage`, so nothing from `pkgs/default.nix`